apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -217,7 +217,40 @@ View:
                                   ]
                               }
                           }
-
+        - Markdown:
+            text: "## DataGrids"
+        - DataGrid:
+            id: "myDataGridExample"
+            allowSelection: true
+            styles:
+              headerStyle:
+                backgroundColor: "white"
+                fontSize: "12px"
+                textColor: "#747184"
+                hasDivider: true
+                borderBottom: false
+            DataColumns:
+              - label: PATIENT
+              - label: ADVOCATE
+            item-template:
+              data:
+                [
+                  { key: "1", patient: "John", advocate: "Mike Ross" },
+                  { key: "2", patient: "Reuben", advocate: "Harvey Dent" },
+                ]
+              name: dummyData
+              template:
+                DataRow:
+                  onRowsSelected:
+                    executeCode:  |
+                      console.log("data", selectedRows, selectedRowKeys);
```

Comment:
@sibteali786  Can we move this part outside of the item-template or template ?

Reply:
makes sense, let me do that